redhat-ipaas-dev-single-tenant.yml

```diff
@@ -1520,5 +1524,5 @@ objects:
         "auth-server-url": "https://${ROUTE_HOSTNAME}/auth",
         "ssl-required": "external",
         "resource": "ipaas-rest",
-        "allow-any-hostname" : true
+        "allow-any-hostname" : "${KEYCLOAK_ALLOW_ANY_HOSTNAME}"
```

Comment:
This shouldn't be quoted (it's a boolean in JSON, even though it's a string parameter in the template - confusing I know...).